### DISCUSSION TITLE
Welcome to supabase_playground Discussions!

### DISCUSSION THREAD

**Discussion by ibhavikmakwana** [ibhavikmakwana/supabase_playground#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TechieBlossom** [comment]
Hi, I am Prateek Sharma, Founder of YouTube channel [Techie Blossom](https://www.youtube.com/c/TechieBlossom).
I create video tutorials and write articles to explain Flutter concepts, UIs, State Management techniques.

**Comment by ibhavikmakwana** [comment]
Restarting this project.
